op-e2e/actions/kromampt_fork_test.go

```diff
@@ -0,0 +1,182 @@
+package actions
+
+import (
+	"context"
+	"testing"
+
+	oppredeploys "github.com/kroma-network/kroma/op-bindings/predeploys"
```

Comment:
Also please sort the imports!
```suggestion
	oppredeploys "github.com/ethereum-optimism/optimism/op-bindings/predeploys"
```